teacher.rb

```diff
@@ -1,8 +1,8 @@
 class Teacher < Person
   attr_accessor :specialisation
 
-  def initialize(specialisation)
-    super()
+  def initialize(specialisation, age)
+    super(age, 'Unknown')
```

Comment:
- [ ] You have done great to apply the changes, however the `'Unknown'` argument passed is not defined. Kindly consider using the correct arguments defined. For example `name` would be appropriate to use for this case ☺️.